certif/tests/integration/components/session-finalization/completed-reports-information-step_test.js

```diff
@@ -188,4 +188,58 @@ module('Integration | Component | SessionFinalization::CompletedReportsInformati
     // then
     assert.dom(screen.getByRole('table', { name: 'Certification(s) terminée(s)' })).exists();
   });
+
+  module('when isEndTestScreenRemovalEnabled feature toggle is disabled', function() {
+    test('it shows the "Écran de fin de test vu" column', async function(assert) {
+      // given
+      this.certificationReports = [run(() => store.createRecord('certification-report', {
```

Comment:
comments: le run n'est pas utile
```suggestion
      this.certificationReports = [store.createRecord('certification-report', {
```